### ISSUE TITLE
`group_code_field_history/code` field being duplicated in editor

### ISSUE THREAD

**Issue by brianwdavis** [precision-sustainable-ag/etl-forms#74]
I suspect this is a bug in the script to generate the list of editable fields, not actually a front-end error. This is the farm history form.

![image](https://user-images.githubusercontent.com/13338560/207959924-a6e923dc-3290-46a4-93e5-56f51c679b56.png)

**Comment by mikahpinegar** [comment]
yeah it is a problem with the script to generate editable fields, i'll resolve it
